src/utils/output.ts

```diff
@@ -47,6 +47,12 @@ function table(
   CliUx.ux.table(responses as unknown as Array<Record<string, unknown>>, columns, options ?? {});
 }
 
+function colorStatus(status: RequestStatus): string {
+  if (status === RequestStatus.Succeeded) return green(status);
```

Comment:
@mdonnalley @mshanemc While working on message formats with Matt, I created the constant [StandardColors](https://github.com/salesforcecli/sf-plugins-core/blob/main/src/sfCommand.ts#L31) in SfCommand. I would recommend using those for coloring output.

I also added a new functiuon on SfCommand, logSuccess(msg:string).

Reply:
cool!  I'm gonna update my scratch create commands with those.